sphinx_reredirects/__init__.py

```diff
@@ -133,6 +138,20 @@ def create_redirects(self, to_be_redirected: Mapping[str, str]) -> None:
 
             self._create_redirect_file(redirect_file_abs, target)
 
+    @staticmethod
+    def _to_relative_path(source: str, target: str) -> str:
+        """Convert any path (relative or absolute) to a relative path"""
+        if not target.startswith(SEP):
+            # Already a relative path
+            return target
+
+        # Prepend target with ".." for each level of nesting in source.
+        print(f"Detecting absolute path: {target} from {source}")
```

Comment:
The two print() should be log debug messages:

```suggestion
        logger.debug(f"Detecting absolute path: {target} from {source}")
```